src/Input/Input.variants.js

```diff
@@ -81,7 +81,10 @@ const DISABLED_STYLE = {
   ...DEFAULT_DISABLED_STYLE,
   backgroundColor: 'backgroundGray',
   borderColor: 'inputGray',
-  color: 'textInputDisabled'
+  color: 'textInputDisabled',
+  ':hover': {
+    borderColor: 'inputGray'
+  }
```

Comment:
The default behaviour for a disabled field is not to have any mouse events, including hover (hovering over a disabled field should not indicate a possible click operation).